### PR TITLE
delete useless params containerized

### PR DESCRIPTION
**Pull request by jiulongzaitian**
Signed-off-by: zhangjie <zhangjie0619@yeah.net>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
     delete useless params containerized
```

### CODE REVIEW

File: cmd/kubelet/app/server.go [hunk 1]
```diff
@@ -714,7 +714,6 @@ func RunKubelet(kubeFlags *options.KubeletFlags, kubeCfg *kubeletconfiginternal.
 		kubeFlags.RegisterNode,
 		kubeFlags.RegisterWithTaints,
 		kubeFlags.AllowedUnsafeSysctls,
-		kubeFlags.Containerized,
```
Comment (dixudx) [inline cmd/kubelet/app/server.go]: This flag is useful. It is used to indicate that `kubelet` is running in containerized mode.

**Review by dixudx — CHANGES_REQUESTED** [review]
`Containerized` is not useless.

Object to this code change.

File: cmd/kubelet/app/server.go (hunk 1 above)
Comment (ncdc) [reply cmd/kubelet/app/server.go]: It appears that this particular boolean in this particular code path is not used. This PR is not removing the code that configures the NsenterMounter, for example (https://github.com/kubernetes/kubernetes/blob/d1e711a6af5379f48be4ea585911977000be2af3/cmd/kubelet/app/server.go#L141-L145).

Comment (yujuhong) [reply cmd/kubelet/app/server.go]: I think @ncdc is right. This doesn't remove the flag, nor the field in the kubelet configuration. It only removes the redundant passing of this variable.

Comment (dixudx) [reply cmd/kubelet/app/server.go]: Thanks @ncdc @yujuhong for pointing out. My bad.

**Pull request merged**